src/pid/status.rs

```diff
@@ -192,6 +192,9 @@ named!(parse_status_state<State>,
           | tag!("Z (zombie)") => { |_| State::Zombie }));
 
 named!(parse_command<String>,   delimited!(tag!("Name:\t"),      parse_line,         line_ending));
+#[cfg(any(all(target_os = "android", target_arch = "arm"), target_os = "freebsd", target_os = "macos"))]
+named!(parse_umask<mode_t>,     delimited!(tag!("Umask:\t"),     parse_u16_octal,    line_ending));
```

Comment:
Should `parse_16_octal` not be added to the used parsers?

Reply:
Dunno, these codes are ported from a PR: https://github.com/danburkert/procinfo-rs/pull/32/files#diff-12540dd342b28489ea301f71fcf83038R190-R192

Reply:
Sorry, I guess I get your point now. I indeed missed the import